test/pummel/test-heapdump-zlib.js

```diff
@@ -1,17 +1,28 @@
 // Flags: --expose-internals
 'use strict';
-require('../common');
+const common = require('../common');
 const { validateSnapshotNodes } = require('../common/heap');
 const zlib = require('zlib');
 
 validateSnapshotNodes('Node / ZlibStream', []);
-// eslint-disable-next-line no-unused-vars
-const gunzip = zlib.createGunzip();
+
+const gzip = zlib.createGzip();
 validateSnapshotNodes('Node / ZlibStream', [
   {
     children: [
-      { node_name: 'Zlib', edge_name: 'wrapped' },
-      { node_name: 'Node / zlib_memory', edge_name: 'zlib_memory' }
+      { node_name: 'Zlib', edge_name: 'wrapped' }
+      // zlib memory is initialized lazily
```

Comment:
Linter complained because it doesn't like comments that start with a lowercase letter. I wouldn't mind seeing that lint rule disabled. But in the meantime, maybe fix by removing the comment? Or rewording as `Memory in zlib is initialized lazily.`? Or (ugh) adding on the previous line `// eslint-disable-next-line capitalized-comments`?

Reply:
Fixed it by expanding the comment.

Reply:
Forgot to run the linter. My bad. 😥

Thanks for fixing the comment!